bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/I18nProviderImpl.java

```diff
@@ -153,13 +158,17 @@ private void setLocation(final String location) {
         }
     }
 
-    private void setTimeZone(final ZoneId timeZone) {
-        if (timeZone != null) {
+    private void setTimeZone(final String zoneId) {
+        if (StringUtils.isBlank(zoneId)) {
+            timeZone = TimeZone.getDefault().toZoneId();
+            logger.warn("No time zone set, fallback to the system time zone '{}'.", timeZone.toString());
```

Comment:
Isn't that the normal situation (as by default nobody should have to set a timezone)?
If it is, I would expect no logging at all (as all is fine), so you might want to set it to debug instead.

Reply:
should the other logging messages (in case no config is provided) be set to debug too?

Reply:
Which other? https://github.com/eclipse/smarthome/pull/4427/files/6e5ca75838d15bc039067f2c3dc2e4ce634d51af#diff-89c1086067d170cd4b631937eef30418L106 is already debug.
And https://github.com/eclipse/smarthome/pull/4427/files/6e5ca75838d15bc039067f2c3dc2e4ce634d51af#diff-89c1086067d170cd4b631937eef30418R143 is a summary - so it might make sense to actually add the time zone information in there.